index.js

```diff
@@ -36,7 +34,7 @@ module.exports = {
     );
 
     if (app.tests) {
-      let type = shouldExcludeAxeCore ? { type: 'test' } : undefined;
+      let type = options.excludeAxeCore ? { type: 'test' } : undefined;
```

Comment:
I think this option is probably defunct now and should be removed?

Reply:
Ya, I wasn't sure why it was there. I can remove.